distri/sqlitebrowser.desktop.appdata.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>sqlitebrowser.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MPL-2.0 and GPL-3.0+</project_license>
+  <name>DB Browser for SQLite</name>
+  <summary>DB Browser for SQLite is a light GUI editor for SQLite databases</summary>
+  <description>
+    <p>DB Browser for SQLite is a high quality, visual, open source tool to create, design, and edit database files compatible with SQLite.</p>
+    <p>It is for users and developers wanting to create databases, search, and edit data. It uses a familiar spreadsheet-like interface, and you don't need to learn complicated SQL commands.</p>
+    <p>Controls and wizards are available for users to:</p>
+    <ul>
+      <li>Create and compact database files</li>
+      <li>Create, define, modify and delete tables</li>
+      <li>Create, define and delete indexes</li>
+      <li>Browse, edit, add and delete records</li>
+      <li>Search records</li>
+      <li>Import and export records as text</li>
+      <li>Import and export tables from/to CSV files</li>
+      <li>Import and export databases from/to SQL dump files</li>
+      <li>Issue SQL queries and inspect the results</li>
+      <li>Examine a log of all SQL commands issued by the application</li>
+    </ul>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+    	<image>https://github.com/sqlitebrowser/sqlitebrowser/raw/master/images/sqlitebrowser.png</image>
+    	<caption>DB Browser for SQLite Screenshot</caption>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">http://sqlitebrowser.org/</url>
+  <url>https://github.com/sqlitebrowser/sqlitebrowser</url>
+  <updatecontact>info@sqlitebrowser.org</updatecontact>
```

Comment:
Hmmm, we don't have an email address set up for the @sqlitebrowser.org domain, so that's probably not right.

Can we point at the GitHub Issues page for the contact point, or leave out the email?

Although we **could** create a working email address via MX record stuff (not hard), I'd rather not for now... it'd be just another target for spammers. :frowning: